packages/web-app-files/src/services/folder/loaderSpace.ts

```diff
@@ -88,6 +88,12 @@ export class FolderLoaderSpace implements FolderLoader {
           }
         }
 
+        yield store.dispatch('Files/loadAncestorMetaData', {
```

Comment:
I'm curious if we can omit the `yield` here. Please try with and without in your share indicators refactoring PR. In theory Vue3 should be much more powerful in updating the file list with the share indicators after initial rendering than Vue2 was.